### ISSUE TITLE
Could not get unknown property 'FileUtils' for object of type org.gradle.process.internal.DefaultExecAction_Decorated

### ISSUE THREAD

**Issue by davidcian** [beryx/badass-jlink-plugin#136]
When running the jpackage task after the jpackageImage task, with imageDir, imageOutputDir, installerOutputDir and installerType set, the execution fails with error message:
*Could not get unknown property 'FileUtils' for object of type org.gradle.process.internal.DefaultExecAction_Decorated*

**Comment by siordache** [comment]
Which version of Gradle do you use? And which JDK?

**Comment by davidcian** [comment]
These are the details from running the `gradle --version` command:

```
Gradle 6.3

Kotlin:       1.3.70                                                             
Groovy:       2.5.10                                                             
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019         
JVM:          14.0.1 (Oracle Corporation 14.0.1+7)                               
OS:           Windows 10 10.0 amd64
```

**Comment by siordache** [comment]
Thanks! I fixed it in 2.17.8.
